Source/init.cpp

```diff
@@ -1,6 +1,7 @@
 #include "diablo.h"
 #include "../3rdParty/Storm/Source/storm.h"
 #include "../DiabloUI/diabloui.h"
+#include <SDL2/SDL.h>

```

Comment:
This should be `#include <SDL.h>` for both SDL2 and SDL1.

Reply:
Unfortunately nothing compiles when I do that :( Not really sure why SDL isn't exposed to things in Source?

Reply:
I think it's because SDL is only linked with the devilutionx target but not devilution:

https://github.com/diasurgical/devilutionX/blob/master/CMakeLists.txt#L241-L250

Reply:
Tried that also :) I might just make a single wrapper function for now, once we can get rid of all the abstraction we can lose a lot of the separation like the dvl namespace and practically the whole SourceS folder. It's all just there to avoid conflicts on Windows.

Reply:
Sent #276